src/main/java/uk/gov/companieshouse/web/accounts/service/companyaccounts/impl/CompanyAccountsServiceImpl.java

```diff
@@ -34,7 +34,7 @@ public String createCompanyAccounts(String transactionId) throws ServiceExceptio
         String uri = CREATE_COMPANY_ACCOUNTS_URI.expand(transactionId).toString();
 
         try {
-            companyAccounts = apiClient.companyAccounts().create(uri, companyAccounts).execute();
+            companyAccounts = apiClient.companyAccounts().create(uri, companyAccounts).execute().getData();
```

Comment:
Do we need to getData() on a create?

Reply:
We do in this case because we pull back the response body and query the self link to fetch the `companyAccountsId`

Reply:
Cool, just wanted to check as it was different to the others 👍